internal/apiserver/server.go

```diff
@@ -61,6 +65,7 @@ type apiServer struct {
 	maxFilterSkip      uint64
 	apiTimeout         time.Duration
 	apiMaxTimeout      time.Duration
+	metricsRegistery   *prometheus.Registry
```

Comment:
Long-term this needs to be stored in a global(ish) location so that other parts of the codebase can use it to register custom metrics.

Storing it in the `apiServer` right now so that it can be reused between the REST and admin router, but it feels wrong. If theres a better place for it or if it needs to be stored as a separate module (maybe `internal/metrics`?) happy to do it just need some guidance / examples of how best to organize this.